bug/op_set_metadata.go

```diff
@@ -25,6 +25,8 @@ func (op *SetMetadataOperation) Hash() (git.Hash, error) {
 }
 
 func (op *SetMetadataOperation) Apply(snapshot *Snapshot) {
+	snapshot.addActor(op.Author)
```

Comment:
Adding metadata to an operation is typically done by the importers/exporters. I'd say it doesn't qualify as being an actor.